### PR TITLE
Improve code style of api.ts file

### PR DESCRIPTION
**Pull request by xuesichao**
**Issue #:**

**Description of changes:**
Use `new URLSearchParams(params)` instead of "hard-coded" query parameters in `api.ts` to improve readability.

**Testing**

1. How did you test these changes?
Run the meeting demo and make sure the updated `joinMeeting`, `endMeeting`, `getAttendee` API work well like before.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
meeting demo.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/meeting/src/utils/api.ts [hunk 1]
```diff
@@ -11,23 +11,26 @@ interface MeetingResponse {
     Meeting: any;
   };
 }
+interface GetAttendeeResponse {
+  name: any;
```
Comment (jing-chen1) [inline apps/meeting/src/utils/api.ts:15]: Just wonder if we are improve the code, can we change this `any` to some specific type like `string`?

Comment (xuesichao) [reply apps/meeting/src/utils/api.ts]: Checked the API definition. The type of return data is `string`, so updated `any` to `string`.

File: apps/meeting/src/utils/api.ts [hunk 2]
```diff
@@ -37,12 +40,14 @@ export async function fetchMeeting(
 }
 
 export function createGetAttendeeCallback(meetingId: string) {
-  return async (chimeAttendeeId: string, externalUserId?: string) => {
-    const attendeeUrl = `${BASE_URL}attendee?title=${encodeURIComponent(
-      meetingId
-    )}&attendee=${encodeURIComponent(chimeAttendeeId)}`;
-    const res = await fetch(attendeeUrl, {
-      method: 'GET'
+  return async (chimeAttendeeId: string): Promise<GetAttendeeResponse> => {
```
Comment (jing-chen1) [inline apps/meeting/src/utils/api.ts:43]: Could you restructure this function and make it same as `fetchMeeting` and `endMeeting`?

Comment (xuesichao) [reply apps/meeting/src/utils/api.ts]: Done

**Review by jing-chen1 — APPROVED** [review]

**Pull request merged**